docs/rbac.rst

```diff
@@ -0,0 +1,269 @@
+Role Based Access Control
+=========================
+
+.. important::
+
+    This feature is a Tech Preview and can be subject of change.
+
+.. important::
+
+    Using user ``admin`` will bypass all RBAC checks.
+
+
+* `Permission` : Permission to do **one** action on one `viewset` as a `view` or a `create`.
+
+  * `Model Permission` : a permission for a model type, it means you can do this action on any object of the model
+
+    * E.g. add (create) which doesn't make sense for object if already exists.
+
+  * `Object Permission` : a permission for `one specific object` only
+
+    * E.g. update or delete. As such is applied for specific content.
+
+* `Role` : Group of permission.
+
+
+RPM plugin presents Role Based Access for these viewsets :
+
+* RPM Alternate Content Source (``/pulp/api/v3/acs/rpm/rpm/``)
+* RPM Distributions (``/pulp/api/v3/distributions/rpm/rpm/``)
+* RPM Publications (``/pulp/api/v3/publications/rpm/rpm/``)
+* RPM Remotes (``/pulp/api/v3/remotes/rpm/rpm/`` and ``/pulp/api/v3/remotes/rpm/uln``)
+
+  * for both types there is one permission starting with ``rpm.rpmremote_[action]``
+
+* RPM Repository (``/pulp/api/v3/repositories/rpm/rpm/``)
+
+
+Default Roles
+-------------
+
+RPM plugin ships few default roles described bellow.
+
+Viewer Role
+^^^^^^^^^^^
+
+This role contains only ``view`` permission for a `viewset`. Using this permission you can control who can
+view specific content for example a repository.
+
+.. code-block:: shell
+   :caption: You can check the role by pulp-cli
+
+    pulp role show --name "rpm.rpmremote_viewer"
+
+.. code-block:: shell
+
+    {
+      "pulp_href": "/pulp/api/v3/roles/62baf888-09c2-4c18-8d3a-354d9c6bdb48/",
+      "pulp_created": "2022-04-07T12:41:11.065567Z",
+      "name": "rpm.rpmremote_viewer",
+      "description": null,
+      "permissions": [
+        "rpm.view_rpmremote"
+      ],
+      "locked": true
+    }
+
+Creator Role
+^^^^^^^^^^^^
+
+This role contain only ``add`` permission for a specific `viewset` to allow user create a new object of its type.
+Be aware that it is possible you will need additional permissions to be able create new object.
```

Comment:
"to be able create new object" should be switched to:

"to be able to create a new object" or "to be able to create new objects"